client/src/i18n/fr/form.json

```diff
@@ -21,7 +21,7 @@
             "CONFIGURE_PAYMENT": "Configurer pour le Paiement",
             "CONFIRM": "Confirmer",
             "CONFIRM_CANCEL": "Confirmer Annulation",
-            "CONNECTION": "Connexion",
+            "CONNECTION": "Connecter",
```

Comment:
In good French, it is: `Se connecter`

Reply:
Thanks!  I'm fixing it.

Reply:
@mbayopanda Just pushed the fix.  Please verify  when you can and then tell Bors to release this tiny fix.